lib/philomena_web/views/layout_view.ex

```diff
@@ -69,10 +69,34 @@ defmodule PhilomenaWeb.LayoutView do
     Config.get(:footer)
   end
 
-  def stylesheet_path(%{theme: "dark"}), do: ~p"/css/dark.css"
-  def stylesheet_path(%{theme: "red"}), do: ~p"/css/red.css"
-  def stylesheet_path(_user), do: ~p"/css/default.css"
-  def dark_stylesheet_path, do: ~p"/css/dark.css"
+  def stylesheet_path(conn, %{theme: theme})
```

Comment:
If you use the above suggestion for the `User` schema, then this can be written like
```elixir
  @themes User.themes()

  def stylesheet_path(conn, %{theme: theme}) when theme in @themes,
    do: # ...
```